scripts/setup.sh

```
@@ -0,0 +1,431 @@
+# Usage:
+#   curl -fsSL https://raw.githubusercontent.com/EcomTree/runpod-comfyui-cloud/main/scripts/setup.sh | bash
+#
+# Version: 2.0 (Optimized for Codex)
+#
+
+set -euo pipefail
+
+# Script configuration
+SCRIPT_VERSION="2.0"
+PROJECT_NAME="runpod-comfyui-cloud"
+REPO_URL="https://github.com/EcomTree/runpod-comfyui-cloud.git"
```

Shellcheck (0.11.0)

[warning] REPO_URL appears unused. Verify use (or export if used externally).

(SC2034)